src/containers/about-us/about-us.container.jsx

```diff
@@ -1,9 +1,69 @@
 /* @flow */
 import React, { Component } from "react";
+import AppBar from "@material-ui/core/AppBar";
+
+import GoogleMapsContainer from "./map/maps";
+import SimpleMediaCard from "./cards/cards";
+
+import "./about-us.css";
+
+var images = [
+  "1.jpg",
+  "2.jpg",
+  "3.jpg",
+  "4.jpg",
+  "5.jpg",
+  "6.jpg",
+  "7.jpg",
+  "8.jpg",
+  "migro.jpg"
+];
 
 class AboutUsContainer extends Component<{}> {
+  
+  renderImage = (imageUrl) => {
```

Comment:
let's create a react small comp for that (instead of a function) like
const AboutUSImage = (props: Props) => <div><img.../></div>;